talks/de.json

```diff
@@ -1,126 +1,151 @@
 {
 	"respondTimeRemaining": {
 		"default": [
-			"Es sind noch {0} bis zu deinem {1}",
-			"Einen Moment bitte. Du hast noch {0} übrig auf dem {1}",
-			"Insgesamt hast du noch {0} auf dem {1}",
-			"Für den {1} verbleiben noch {0}. Keine Sorge, ich halet dich auf dem laufenden."
+			"Es gibt {0} Zeit links auf dieser {1}",
+			"Bitte halten Sie die Linie, während ich überprüfen. Scheint Sie haben {0} auf die linke {1}",
+			"Sie haben eine Gesamtsumme von {0} auf die linke {1}",
+			"Für die {1} Sie haben {0} nach links zu gehen. Keine Sorge werde ich halte Sie auf dem Laufenden"
 		],
 		"short": [
 			"{0}",
-			"Der {1} ist bei {0}"
+			"Die {1} hat {0} links"
 		]
 	},
 	"respondDeleteAll": {
 		"default": [
-			"Keine sorge ich habe die {} für dich gelöscht.",
-			"Alles erledigt, ich habe deine {} gelöscht.",
-			"Ok, ich habe alle {} gelöscht",
-			"{} gelöscht."
+			"Keine Sorge, ich habe die {} s für Sie gelöscht",
+			"Alle sortiert, gereinigt, alle Ihre {} ist wie niemand hier war",
+			"Die Tische sauber Kapitän. Nicht mehr {} s gespeichert",
+			"{} S gelöscht. Hausarbeit erledigt. Ich glaube, meine Widerstände gehen würde polieren jetzt"
 		],
 		"short": [
-			"Erledigt.",
-			"Ok."
+			"Sortiert",
+			"Erledigt"
 		]
 	},
 	"respondDelete": {
 		"default": [
-			"Ok, ich habe den {} für dich gelöscht.",
-			"Erledigt, ich habe deinen {} gelöscht.",
-			"Von dem {} wirst du nichts mehr hören.",
-			"Der {} ist fort. Ok?"
+			"Keine Sorge habe ich gelöscht, dass {} für Sie.",
+			"Alle sortiert, Ihre {} ist wie niemand hier war aufgeräumt.",
+			"Wow, Ich löschte nicht nur, dass {} i es zum Mars auf eine Mission geschickt.",
+			"Scheint tat ich das richtig. Die {} ist jetzt nicht da."
 		],
 		"short": [
-			"Erledigt.",
-			"OK."
+			"Sortiert",
+			"Erledigt"
 		]
 	},
 	"respondFoodTimer": {
 		"default": [
-			"Hey chef, wir sind halb durch den Timer. Zeit nach dem Essen zu schauen!",
-			"Aufgepasst. Du solltest mal schauen wie es mit dem Essen aussieht.",
-			"Riecht das hier verbrannt? Sollte doch gerade mal die hälfte der Zeit sein?",
-			"Ich bekomme langsam hunger, wie weit ist das Essen?"
+			"Hey Chef, wir sind auf halbem Weg durch den Timer. Zeit auf Ihrem Essen zu überprüfen!",
+			"Achtung Koch. Schauen Sie auf Ihr Essen im Falle muss es gerührt oder gedreht.",
+			"Tippen Sie auf, tippen Sie auf. <Pause = „250ms“ /> weiß nicht, ob dieses Mikrofon funktioniert. <Pause = „150ms“ /> Ummmm Achtung Käufer, foodcheck auf Insel 3 bitte",
```

Comment:
yeah, the fun lines get a bit (completely) broken in German I guess - maybe there is just no humor in German :P